_src/static/sass/utilities/_variables.scss

```diff
@@ -362,6 +363,12 @@ $headings-primary-color: palette(grayscale, x-dark);
 $headings-secondary-font-family: $font-family-sans-serif;
 $headings-secondary-color: palette(grayscale, base);
 
+// buttons
```

Comment:
@talbs Aren't these done in the `_buttons.scss` file, which institutions would then overwrite. I figured the palettes (and some other configuration things) are in `_variables.scss` but that styles specific to buttons would be in that respective partial. Let me know as I don't think it's good to have button configurations in two different places.

`_variables.scss` has global button configuration, such as border style, shading, radius
`_buttons.scss` has specific button configuration, such as background/foreground color, border color, etc.

This okay with you?